tests/unittests/benchmark.json

```diff
@@ -1,7 +1,7 @@
 {
   "time": {
-    "initialize": 0.295,
-    "run": 1.124
+    "initialize": 0.548,
+    "run": 3.307
```

Comment:
Oh yikes...this is now 5x slower than it was a few days ago :|

Reply:
This is running on very small VM in docker on mac, this will affect result as well. If you want more real number, you should run it on your machine